Explorer/Assets/Scripts/SceneRunner/SceneFacade.cs

```diff
@@ -86,9 +87,23 @@ public async UniTask StartUpdateLoopAsync(int targetFPS, CancellationToken ct)
 
             SetTargetFPS(targetFPS);
 
-            // Start the scene
+            try
+            {
+                // Start the scene
+                await runtimeInstance.StartScene();
+            }
+            catch (OperationCanceledException) { return; }
+            catch (ScriptEngineException e)
+            {
+                sceneExceptionsHandler.OnJavaScriptException(e);
+                return;
+            }
+            catch (Exception e)
```

Comment:
It should be caught in `EngineApiWrapper`: it's not caught there because `RethrowSceneExceptionsHandler` is used instead of the correct one: please fix it, it will reduce the noise in this class

Reply:
Also, please wrap in try catch 
```
[UsedImplicitly]
        public ScriptableByteArray CrdtGetState()
        {
            ArraySegment<byte> result = api.CrdtGetState();
            return result.Count > 0 ? new ScriptableByteArray(result) : ScriptableByteArray.EMPTY;
        }
```
as well. Similarly to `CrdtSendToRenderer`, it will allow to get rid of this `catch` fully

Reply:
I meant the wrong exception handler is used in `EngineApiWrapper`.
This is what I meant https://github.com/decentraland/unity-explorer/pull/176/commits/997ca44567c07d95360d663f0122b272c55e84c2

Can I ask you to check if everything is fine after my change?

Reply:
Thanks for the change. Everything seems to be working as expected